code/modules/mining/mine_items.dm

```diff
@@ -2,10 +2,16 @@
 
 //this item is intended to give the effect of entering the mine, so that light gradually fades
 /obj/effect/light_emitter
-	name = "Light-emtter"
+	name = "Light emtter"
 	anchored = 1
+	invisibility = 101
 	unacidable = 1
-	light_range = 8
+	var/set_light = 8
+	var/set_cap = 0
+	
+/obj/effect/light_emitter/New()
+	..()
+	set_light(set_light, set_cap)
```

Comment:
You won't need this; the second argument; `set_cap` is the `light_power` (intensity) of light; it's an atom level var, so you can just have `light_range = 8` and `light_power = 0`